android/package.json

```diff
@@ -15,13 +15,13 @@
 	],
 	"minSDKVersion": "16",
 	"vendorDependencies": {
-		"android sdk": ">=23 <=25.x",
-		"android build tools": ">=23 <=25.x",
-		"android platform tools": ">=17 <=25.x",
+		"android sdk": ">=25 <=26.x",
```

Comment:
Ah, I'll update the PR.

I guess we could leave it at `23`, I wanted to keep it inline with our support library.